### ISSUE TITLE
Treat `13584-*` as separate documents from iso-10303-x

### ISSUE THREAD

**Issue by ronaldtse** [metanorma/stepmod2mn#146]
Let's not have `iso-13584*` documents in the SRL because it is supposed to be separate.

Can we skip the migration of them?

**Comment by stuartgalt** [comment]
I think that the SRL should not have a PDF version of  13584 but I believe some of the other parts may reference it.  I defer to Tom.

**Comment by ronaldtse** [comment]
@stuartgalt we could have an 13584 as part of the HTML collection. It's just that I remembered somewhere Tom said remove them from SRL?

**Comment by TRThurman** [comment]
@ronaldtse @opoudjis @stuartgalt 
WG12 decided to keep 13584-20 in the SRL.
Use the 13584-20 from the detached docs for the source.

**Comment by TRThurman** [comment]
@ronaldtse 
Can we close this?

**Comment by ronaldtse** [comment]
Yes we can now! But we did not validate the PDF of 13584-20. Let's make that a new task.
